storage/minio/package.json

```diff
@@ -34,7 +34,9 @@
     "test:unit:frontend": "cypress run --browser chrome --config-file node_modules/@itwin/object-storage-common-config/s3.unit.cypress.config.ts",
     "test:unit:frontend:debug": "cypress open --browser chrome --config-file node_modules/@itwin/object-storage-common-config/s3.unit.cypress.config.ts",
     "test:integration:backend": "npm run test:integration:downloadMinio && npm-run-all -p -r test:integration:startMinio test:integration:backend:runTests",
-    "test:integration:backend:runTests": "wait-on tcp:9000 -t 30000 && node lib/test/integration/backend/RunIntegrationTests.js",
+    "test:integration:backend:runTests": "wait-on tcp:9000 -t 30000 && npm run test:integration:backend:runTests:common && npm run test:integration:backend:runTests:local",
+    "test:integration:backend:runTests:local": "mocha lib/test/integration/backend/**/*.test.js --color --exit -t 30000",
```

Comment:
I think better naming for the sub-scripts would be `common` and `azureSpecific` (or `ossSpecific` or `minioSpecific`). `local` is pretty generic and I am not sure it fits the best in this context.

Reply:
to match that maybe the test suites to could be renamed `MinioServerStorage specific tests`?
![image](https://github.com/iTwin/object-storage/assets/70565417/bf00ef50-2a02-4792-9765-957a9e8184a5)
not sure about this one though

Reply:
Renamed scripts. I disagree about the test names. Specific tests doesn't really describe anything. Keeping it as `internal tests` let's us understand that these tests use methods which are not exposed to users.